scss/styles.scss

```diff
@@ -21,3 +23,86 @@
 .wolox-logo {
   width: 200px;
 }
+
+.main-section {
+  margin: 0 140px;
+}
+
+.technology-group {
+	margin: 100px 0 40px 0;
```

Comment:
If you give them margin up, the first one should not have it. You could do this with the pseudo class `: first-child` or`: not (: first-child) `.
And also, there is a shorter way to write `margin: 100px 0 40px 0;`: `margin: 100px 0 40px;`

Reply:
Another comment, it is better than the separation of the main content with the navbar, as determined by the more general container and not by one of the subcontainers.